torch/CMakeLists.txt

```diff
@@ -465,3 +465,129 @@ install(FILES
     ${PROJECT_BINARY_DIR}/TorchConfigVersion.cmake
     ${PROJECT_BINARY_DIR}/TorchConfig.cmake
     DESTINATION share/cmake/Torch)
+
+set(TORCH_PYTHON_SRCS
+  ${TORCH_SRC_DIR}/csrc/autograd/anomaly_mode.cpp
```

Comment:
${TORCH_SRC_DIR} -> ${CMAKE_CURRENT_SOURCE_DIR}

Reply:
TORCH_SRC_DIR is consistent with the rest of the file - would you like me to change it throughout?